### ISSUE TITLE
Applying migrations

### ISSUE THREAD

**Issue by io-ma** [Dmitri-Sintsov/django-organizations-permissions#1]
Hi, 
Thank you for creating this repo!
I included it in my Django app :-)
I am trying to run "`makemigrations`" but without success.
Migrations for 'org_permissions':                                                              │
 ```
 /usr/local/lib/python3.6/site-packages/org_permissions/migrations/0001_initial.py            │
    - Create model OrganizationPermission 
```
Then I do `migrate` and get `CommandError: App 'org_permissions' does not have migrations.`
If I access add organization permissions in my admin area I get:
`ProgrammingError: relation "org_permissions_organizationpermission" does not exist`
My settings.py:
https://pastebin.com/bD90G1mv

**Comment by Dmitri-Sintsov** [comment]
Please try:

    python manage.py makemigrations org_permissions
    python manage.py migrate org_permissions

**Comment by io-ma** [comment]
Tried it, it doesn't work, I think the way I have my auth setup conflicts with org_permissions.auth_backends.OrganizationModelBackend
I am modifying my settings and come back to you to let you know how it goes.
Ok, coming back, I changed settings to include org_permissions.permissions + the backend.
Doesn't want to work. Can I mail you? Or I could give you access to the code, is it ok?

![Desktop screenshot](https://user-images.githubusercontent.com/2306628/72152640-469c4e00-33b4-11ea-9523-d016e648b93c.png)

Here's the new settings.py:

https://pastebin.com/vPkY2rrC
I reset the migrations, dropped database, recreated everything, same thing, migrations to org_permissions don't apply.
